### PR TITLE
Add v0.6 announcement post

### PR DESCRIPTION
**Pull request by StanczakDominik**
Here's a basic draft :) I'm going to crosspost it to my personal blog later, if you don't mind.

- [ ] add an announcement on the main page, like we did for Thomson

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Giles doesn't seem to be cooperating, but this is available https://256-105815475-gh.circle-artifacts.com/0/output/posts/plasmapy-v060-release/index.html.

File: posts/plasmapy-v060-release.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+For this year, we're planning to have a roughly 3-month cadence for releases. Thus,
+the next release will come by mid June.
+
+It should contain a refactored particle tracker, support for neoclassical
+transport coefficient calculations for axisymmetric devices along with our
+first forays into tools for fusion.
+
+Also, for the first time in the history of PlasmaPy, we've got [a roadmap for
+the year](https://hackmd.io/@plasmapy/ry0mmnj6v/edit).
```
Comment (namurphy) [inline posts/plasmapy-v060-release.md:87]:
```suggestion
To learn more about our development plans, check out [PlasmaPy's
development roadmap](https://hackmd.io/@plasmapy/ry0mmnj6v/edit)
for the year.
```
...though do we want to link to an editable document here?

**Review by namurphy — DISMISSED** [review]
I have a few suggestions but I don't think it needs much more work.  Thank you for writing this!

**Comment by namurphy** [comment]
> I'm going to crosspost it to my personal blog later, if you don't mind.

Please do!  You wrote it and in any case it's CC BY 4.0, so it's all good.

File: posts/plasmapy-v060-release.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Analysis & Diagnostic framework; Langmuir probes
+
+<center>
+[![Langmuir probe sweep](https://docs.plasmapy.org/en/stable/_images/notebooks_analysis_swept_langmuir_find_floating_potential_21_0.png)](https://docs.plasmapy.org/en/stable/notebooks/analysis/swept_langmuir/find_floating_potential.html)
+</center>
+
+Erik Everson is spearheading work on our analysis & diagnostic framework, about
+which you can read [here](https://docs.plasmapy.org/en/stable/ad/index.html).
+As the first example, we've got a new and improved set of tools for swept Langmuir
+analysis. Check it out [here](https://docs.plasmapy.org/en/stable/ad/index.html).
```
[the remark this answers is not in the document]
Comment (StanczakDominik) [reply posts/plasmapy-v060-release.md:75]: Thanks! I need to find some ways to more easily keep that in mind while writing...

File: posts/plasmapy-v060-release.md (hunk 1 above)
Comment (StanczakDominik) [reply posts/plasmapy-v060-release.md:87]: Fair point; I'll get rid of this part.

**Pull request merged**